pkg/api/scylla/validation/cluster_validation.go

```diff
@@ -503,9 +503,27 @@ func isRackStatusUpToDate(sc *scyllav1.ScyllaCluster, rack string) bool {
 }
 
 func GetWarningsOnScyllaClusterCreate(sc *scyllav1.ScyllaCluster) []string {
-	return nil
+	var warnings []string
+
+	warnings = append(warnings, warningsForScyllaClusterSpec(&sc.Spec, field.NewPath("spec"))...)
+
+	return warnings
+}
+
+func warningsForScyllaClusterSpec(spec *scyllav1.ScyllaClusterSpec, fldPath *field.Path) []string {
+	var warnings []string
+
+	if len(spec.Sysctls) > 0 {
+		warnings = append(warnings, fmt.Sprintf("%s: deprecated; use nodeConfig.spec.sysctls instead", fldPath.Child("sysctls")))
```

Comment:
`nodeConfig.spec.sysctls` might suggest it's a field in `ScyllaCluster`. Maybe we should be more explicit here and make it clear that `NodeConfig` is another CRD?

Reply:
done